### DISCUSSION TITLE
Dynamic timestamp support in GitHub Flavored Markdown

### DISCUSSION THREAD

**Discussion by pl4nty** [community/community#9412] [Pull Requests]
When writing comments/issues/PRs etc, it would be useful to create timezone-aware timestamps, particularly for distributed teams. 

GitHub already uses [@github/time-elements](https://github.com/github/time-elements) web component extensions in many products. However, they aren't supported in [GitHub Flavored Markdown](https://github.com/github/time-elements), which prevents timezone-aware or relative timestamps. If support was added, it could improve developer experience and workflows across several areas including issues, PR reviews, Actions CI/CD, etc.

**Comment by Fyko** [comment]
Would love to see this feature.

**Comment by colelawrence** [comment]
Something similar to this is Discord's support for this (See https://discordtimestamp.com/).
I'd use this for relative times attached to my Job Step Summaries & automatic Pull Request comments, so it's easier to tell when the deployment was made.

**Comment by ndelangen** [comment]
This would indeed be a great feature to have

**Comment by burningtnt** [comment]
It would be greate to support this feature.

**Comment by joyliu-q** [comment]
It would be great to support this feature!

**Comment by karolba** [comment]
I'm not sure if it's documented anywhere, but `<relative-time ...>` appears to already work in GitHub Flavored Markdown. I ran across it in this comment https://github.com/OpenRouterTeam/awesome-openrouter/pull/135#issuecomment-5544724622 (use the `Copy Markdown` button there)

`<relative-time datetime="2026-09-04T18:20:24.546857Z">2026-09-04T18:20:24.546857Z</relative-time>` produces "<relative-time datetime="2026-09-04T18:20:24.546857Z">2026-09-04T18:20:24.546857Z</relative-time>"

**Reply by svrnm** [reply]
Thanks, this is good to know!

**Reply by Fyko** [reply]
@pl4nty could you mark this as the answer?

**Reply by SwooshyCueb** [reply]
This seems like a partial answer. Is there a solution for absolute timezone/locale-aware timestamps?

**Reply by SwooshyCueb** [reply]
Actually, I was able to find the documentation, and it looks like it can be used for absolute timestamps https://github.github.com/relative-time-element/

**Reply by svrnm** [reply]
What's missing for me, not sure if possible, but we often have things like meeting hours in project READMEs etc, so a

SIG X meets every Thursday at 8am PT

So everyone outside the given timezone needs to do their math, I don't see if/how the element could support that
